cl/disclosures/api_serializers.py

```diff
@@ -74,6 +75,14 @@ class Meta:
         fields = "__all__"
 
 
+class JudgeSerializer(DynamicFieldsMixin, HyperlinkedModelSerializerWithId):
+    positions = PositionSerializer
+
+    class Meta:
+        model = Person
+        exclude = ("race",)
+
+
```

Comment:
Why not use PersonSerializer from people_db.api_serializers? If the answer is "That one has too much junk," then let's create a `SlimPersonSerializer` or something, and put it over in people_db.api_serializer` instead of in the FD serializer. At least that way, they can be side by side.

Reply:
It was about getting the Person data that wasn't just a hyperlinked model.  I really need just a few data points from person, name etc.  My dream is to also bring back court information because I feel like that should be added to the instant results ...

well the dream is for an SVG of the region each judge represents and the court name

Reply:
Gotcha. Well, let's move this serializer over to the people_db, in any case. I'm not sure slimming it makes a huge difference atm though.